mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
```

Comment:
That is plain wrong. This sentence is a pun on the previous one, with talks about tennis. The pun should be preserved somehow, not translated literally.

Reply:
I don't know how to preserve the same meaning and the relation with the previous sentence. The best thing I can come with is
```suggestion
msgstr "El tenis se me da mejor..."
```
that means "I am better at tennis...". @cyphra, @Sanglorian, what do you think?